jobs.py

```diff
@@ -17,14 +18,16 @@ def send_notice_email():
     People = Query()
     target_users = db.search(People.status == 0)
     for user in target_users[:30]:  # 这里最多需要150 * 30s来发送完毕
+        token = uuid.uuid1()
         send_email(
             to_email=user['email'],
             subject='快来领取您的《wuhan2020开源社区志愿者证书》',
             content='感谢您的辛苦付出，请点击链接 <a href="https://community.wuhan2020.org.cn/zh-cn/certification/index.html?token='
-                    + user['token'] + '">https://community.wuhan2020.org.cn/zh-cn/certification/index.html?token='
-                    + user['token'] + '</a>领取您的《志愿者证书》\nwuhan2020 开源社区\n社区网址：<a href="https://community.wuhan2020.org.cn/">https://community.wuhan2020.org.cn/</a>',
+                    + token + '">https://community.wuhan2020.org.cn/zh-cn/certification/index.html?token='
+                    + token + '</a>领取您的《志愿者证书》\nwuhan2020 开源社区\n社区网址：<a href="https://community.wuhan2020.org.cn/">https://community.wuhan2020.org.cn/</a>',
         )
-        update_status(email=user['email'], status=1)
```

Comment:
status should be updated to zero I think

Reply:
Now I use  update_status_and_token ,  after sending mail. it should 1

Reply:
logic probably not correct